connector/protobuf/src/test/scala/org/apache/spark/sql/protobuf/ProtobufFunctionsSuite.scala

```diff
@@ -1136,7 +1208,8 @@ class ProtobufFunctionsSuite extends QueryTest with SharedSparkSession with Prot
           val df = emptyBinaryDF.select(
             from_protobuf_wrapper($"binary", name, descFilePathOpt, options).as("empty_proto")
           )
-        assert(df.schema == structFromDDL("empty_proto struct<>"))
+        assert(df.schema ==
+          structFromDDL("empty_proto struct<>"))
```

Comment:
I thought we would not have empty struct after this PR.

Reply:
This test don't configure the option explicitly, so the default behavior is the same as before.

Reply:
Could you remove this diff. Looks like spurious change.